src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/ServiceHearingValuesMappingTest.java

```diff
@@ -148,13 +153,14 @@ void shouldMapServiceHearingValuesSuccessfully() {
         // given
         SscsCaseData sscsCaseData = sscsCaseDetails.getData();
         // when
-        final ServiceHearingValues serviceHearingValues = ServiceHearingValuesMapper.mapServiceHearingValues(sscsCaseDetails, referenceData);
+        final ServiceHearingValues serviceHearingValues = ServiceHearingValuesMapping.mapServiceHearingValues(sscsCaseDetails, referenceData);
         final HearingWindow expectedHearingWindow = HearingWindow.builder()
-                .hearingWindowDateRange(HearingWindowDateRange.builder().hearingWindowStartDateRange("2022-02-26").build()).build();
+                .hearingWindowDateRange(HearingWindowDateRange.builder()
+                .hearingWindowStartDateRange(LocalDate.now().plusDays(DAYS_TO_ADD_HEARING_WINDOW_TODAY).toString()).build()).build();
         //then
         assertEquals(sscsCaseData.getCaseAccessManagementFields().getCaseNameHmctsInternal(), serviceHearingValues.getCaseName());
         assertEquals(sscsCaseData.getCaseAccessManagementFields().getCaseNamePublic(), serviceHearingValues.getCaseNamePublic());
-        assertTrue(serviceHearingValues.isAutoListFlag()); //
+        assertFalse(serviceHearingValues.isAutoListFlag()); //
```

Comment:
Very minor: Can we get rid of the // at the end of this line?

Reply:
Removed.